### PR TITLE
Add style preserving the table, and breaking the sessions in column b…

### LINKED ISSUE #63797: The User profile Sessions, with responsive problems on the layout.
**What happened**:

As you can see on .gif, the Session table cross the layout if you reduce the screen or use it on mobile.

**How to reproduce it (as minimally and precisely as possible)**:
To reproduce, after the login, access `/profile`, and reduce the screen, you can notice that the session table not rearrange.
Same happens on: `https://<username>.grafana.net/profile`

**What you expected to happen**:
We expected that the table would change the orientation vertically on small screens.

**Anything else we need to know?**:

**Environment**:
- Grafana version: `Grafana v9.5.0-pre`
- Data source type & version: Develop and production
- OS Grafana is installed on: macOS 13.2.1
- User OS & Browser: macOS - Chrome & Android - Chrome
- Grafana plugins: none
- Others: none

### PR DESCRIPTION
**Pull request by fl4viooliveira**
…locks

**What is this feature?**

This feature is a conservative approach to fixing the layout issue.

**Why do we need this feature?**

To the layout didn't break on mobile 

**Who is this feature for?**
This fixes the layout issue on the profile admin for mobile screen size

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #63797

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ashharrison90** [comment]
hi @fl4viooliveira 👋 

thanks for taking a look at this! 🙌 

i find it a little hard to parse the information, not sure this is the best experience for either desktop or mobile tbh 😅
image

i've put together an alternative [here](https://github.com/grafana/grafana/pull/63858). it basically just lets you scroll the table when it overflows the content. this isn't ideal either, but i think it's a good stop-gap whilst we evaluate the best way to display data like this on a mobile.

what do you think? 🤔

**Comment by fl4viooliveira** [comment]
Yes, you are totally right, even if adding more lines in your way will look better.
But I'm happy to try to find a way to wrap the lines or the table cell content.
Thank you for the feedback.

**Comment by ashharrison90** [comment]
tbh i don't think we currently have a consistent design pattern for how to display tabular data like this on smaller screens. in some places we use a media breakpoint and switch to a card layout, but i don't think that will work everywhere.

think it might need a bigger pass through design before deciding on anything more drastic.

**Comment by ashharrison90** [comment]
hi @fl4viooliveira 👋 

closing this in favour of https://github.com/grafana/grafana/pull/63858 (which has just been merged)

we're going to have a bigger conversation internally about how best to handle overflowing tables across grafana and hopefully make some more widespread changes. thanks again for raising this + looking into it 🙌
